universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -60,6 +64,10 @@ public void reset() {
     nextId.set(1L);
   }
 
+  public ImmutableMap<QuestionType, Question> getTypeToQuestionMap() {
```

Comment:
(blocking): this is not a canonical set of questions. It is just a sample of questions that has a question for each question type. 

Can you add a docstring describing what this is, and change the method name to reflect this.

Reply:
One possible name is `getQuestionDefinitionSamplesForAllTypes`.

I'm terrible with names though.

Reply:
Resolved